airsonic-main/src/main/java/org/airsonic/player/service/sonos/SonosServiceRegistration.java

```diff
@@ -31,75 +30,159 @@
 import org.apache.http.impl.client.CloseableHttpClient;
 import org.apache.http.impl.client.HttpClients;
 import org.apache.http.message.BasicNameValuePair;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
 
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
 
 /**
+ * <p>Registration with Sonos controller. They are 2 types of registration they are still supported by Sonos.
+ * The ANONYMOUS and APPLICATION_LINK. The third USER_ID, must still work but will be not supported.</p>
+ * <p></p>
  * @author Sindre Mehus
+ * @author Nacrylic
  * @version $Id$
  */
+@Component
 public class SonosServiceRegistration {
-
     private static final Logger LOG = LoggerFactory.getLogger(SonosServiceRegistration.class);
 
-    public void setEnabled(String airsonicBaseUrl, String sonosControllerIp, boolean enabled, String sonosServiceName, int sonosServiceId) throws IOException {
+    /**
+     * The type of Authentication fo Sonos, the old want USER_ID, is will be not supported. We must use the
+     * Anonymous or AppLink. The USER_ID
```

Comment:
What about something like this instead:

```
The type of Authentication for Sonos: we must either use the Anonymous of Applink method, since the old USER_ID one isn't really supported anymore.
```